File: DCAnalyticsMobile/DCAnalyticsMobile/Models/Training.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DCAnalyticsMobile.Models
{
    public class Training : DataCollectionObject
    {
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Location { get; set; }
        public List<Trainer> Trainers { get; set; }
        public List<Topic> Topics { get; set; }
        public List<Trainee> Trainees { get; set; }
    }
}

File: DCAnalyticsWebShell/ClientApp/src/models/Training.ts
import { Topic } from "./Topic";
import { Trainer } from "./Trainer";

export class Training {
  public OID: number
  public Key: string
  public Name: string
  public StartDate: Date
  public EndDate: Date
  public Location: string
  public Trainers: Trainer[]
  public Topics: Topic[]
  public Trainees: string[]
  public ConfigurationId: number
}
